site/pages/search/index.hbs

```diff
@@ -404,6 +405,15 @@
 &lt;/div&gt;
 </code></pre>
 
+<h3>Advanced Search Link</h3>
+<p>Advanced search link below the pagination tab will allow users to enhance the search result.</p>
+
+<pre><code>
+&lt;div&gt;
+	&lt;p&gt;&lt;a href="https://www.canada.ca/en/sr/srb/sra.html"&gt;Advanced search&lt;/a&gt;&lt;/p&gt;
```

Comment:
Replace this: "https://www.canada.ca/en/sr/srb/sra.html" by "[URL of the advance search page]"